udica/policy.py

```diff
@@ -138,6 +138,74 @@ def create_policy(opts, capabilities, mounts, ports, append_rules):
             policy.write('    (allow process ' + list_ports(item['hostPort'], item['protocol']) + ' ( ' + perms.socket[item['protocol']] + ' (  name_bind ))) \n')
 
     # mounts
+    if inspect_format == "CRI-O":
+        write_policy_for_crio_mounts(mounts, policy)
+    else:
+        write_policy_for_podman_mounts(mounts, policy)
+
+    if append_rules != None:
+        for rule in append_rules:
+            if opts['ContainerName'] in rule[0]:
+                policy.write('    (allow process ' + rule[1] + ' ( ' + rule[2] + ' ( ' + rule[3] + ' ))) \n')
+            else:
+                print('WARNING: process type: ' + rule[0] + ' seems to be unrelated to this container policy. Skipping allow rule.')
+
+    policy.write(')')
+    policy.close()
+
+
+def write_policy_for_crio_mounts(mounts, policy):
+    for item in mounts:
+        if item['hostPath'].startswith('/var/lib/kubelet'):
+            # These should already have the right context
+            continue
+        if item['hostPath'] == LOG_CONTAINER:
+            if item['readonly']:
+                policy.write('    (blockinherit log_container)\n')
+            else:
+                policy.write('    (blockinherit log_rw_container)\n')
+            add_template("log_container")
+            continue
+
+        if item['hostPath'] == HOME_CONTAINER:
+            if item['readonly']:
+                policy.write('    (blockinherit home_container)\n')
+            else:
+                policy.write('    (blockinherit home_rw_container)\n')
+            add_template("home_container")
+            continue
+
+        if item['hostPath'] == TMP_CONTAINER:
+            if item['readonly']:
+                policy.write('    (blockinherit tmp_container)\n')
+            else:
+                policy.write('    (blockinherit tmp_rw_container)\n')
+            add_template("tmp_container")
+            continue
+
+        if item['hostPath'] == CONFIG_CONTAINER:
+            if item['readonly']:
+                policy.write('    (blockinherit config_container)\n')
+            else:
+                policy.write('    (blockinherit config_rw_container)\n')
+            add_template("config_container")
+            continue
+
+        # TODO(jaosorior): Add prefix-dir to path. This way we could call this
```

Comment:
Would be possible to implement this TODO? I'm afraid that we forget about it, if there is just TODO liner in code. :)